### ISSUE TITLE
Update installation guide to reflect new Blazor project structures

### ISSUE THREAD

**Issue by Awide** [vikramlearning/blazorbootstrap#596]
I believe from .NET 8.0 (although at least from the current version) the file templates were updated to a different structure. There is no more index.html and for client and server sided rendering two folders are generated instead of only one.

**Comment by gvreddy04** [comment]
@Awide Thank you for trying BlazorBootsrtap. Please check the documentation at https://docs.blazorbootstrap.com/getting-started/blazor-webassembly-net-8. Let us know if anything is missing.

**Comment by Awide** [comment]
Yes, so the installation guide cannot be used with the newest blazor projects. Just created a webassembly project and immediately got two folders instead of one, one for client and one for the project.
There is no body section of the wwwroot/index.html file because there is no index.html file. There is also no _framework/blazor.webassembly.js reference.

If I do them in App.razor instead, I get these errors when I try running the project.
`/sageapp/sageapp/Components/Layout/MainLayout.razor(26,71): error CS0246: The type or namespace name 'SidebarDataProviderRequest' could not be found (are you missing a using directive or an assembly reference?) [/sageapp/sageapp/sageapp.csproj]
/sageapp/sageapp/Components/Layout/MainLayout.razor(26,24): error CS0246: The type or namespace name 'SidebarDataProviderResult' could not be found (are you missing a using directive or an assembly reference?) [/sageapp/sageapp/sageapp.csproj]
/sageapp/sageapp/Components/Layout/MainLayout.razor(34,25): error CS0246: The type or namespace name 'NavItem' could not be found (are you missing a using directive or an assembly reference?) [/sageapp/sageapp/sageapp.csproj]
/sageapp/sageapp/Components/Layout/MainLayout.razor(23,5): error CS0246: The type or namespace name 'Sidebar' could not be found (are you missing a using directive or an assembly reference?) [/sageapp/sageapp/sageapp.csproj]
/sageapp/sageapp/Components/Layout/MainLayout.razor(24,17): error CS0246: The type or namespace name 'NavItem' could not be found (are you missing a using directive or an assembly reference?) [/sageapp/sageapp/sageapp.csproj]
dotnet watch ⏳ Waiting for a file to change before restarting dotnet...`

**Comment by gvreddy04** [comment]
@Awide I'll take a look tomorrow.

**Comment by Awide** [comment]
> @Awide I'll take a look tomorrow.

Tried making a webassembly project too, I cannot get through manually installing the components. Does it work for you if you start everything from scratch, trying to add manually?

`dotnet new blazor --interactivity Server --auth Individual` etc

**Comment by oleksiivasylenko** [comment]
In net 9, blazor moved forwad. @gvreddy04 Could you please review new blazor structure and modify/add new ready to use template for net 9. So it would be much easier to strart new project.
